internal/process/provisioning/btp_operator_overrides.go

```diff
@@ -29,7 +29,7 @@ func (s *BTPOperatorOverridesStep) Run(operation internal.Operation, log logrus.
 	f := func(op *internal.Operation) {
 		op.InstanceDetails.ServiceManagerClusterID = clusterID
 	}
-	if !operation.InputCreator.Configuration().ContainsAdditionalComponent(internal.BTPOperatorComponentName) {
+	if !operation.InputCreator.Configuration().ContainsAdditionalComponent(internal.BTPOperatorComponentName, log) {
```

Comment:
But if this part is going to be left in code we should be consistent.
So we should pass `log` or `logger` but not both at the same time.

Reply:
Removed.